build-system/tasks/dist.js

```diff
@@ -229,15 +225,6 @@ async function prebuild() {
   await preBuildWebPushPublisherFiles();
 }
 
-/**
- * Copies parsers from the build folder to the dist folder
- */
-async function copyParsers() {
-  const startTime = Date.now();
-  await fs.copy('build/parsers', 'dist/v0');
-  endBuildStep('Copied', 'build/parsers/ to dist/v0', startTime);
-}
-
```

Comment:
Bravo!

Reply:
Yeah....lets see if tests pass 🤞

Reply:
Aha. Looks like this was needed after all. Maybe just revert this file so the diffs go away?

Reply:
definitely. just hacking right now to see if tests will pass